egs/tedlium/s5_r2/conf/segment_long_utts.conf

```diff
@@ -0,0 +1,28 @@
+# TF-IDF similarity search options
+max_words=1000
```

Comment:
Does this config file need to exist?
Why not just have defaults at the top of the script, and only override those that you really need to override?